js/script.js

```diff
@@ -0,0 +1,55 @@
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+function renderBooks() {
+  const listDisplay = document.getElementById('list');
+  listDisplay.innerHTML = '';
+
+  books.forEach((book, index) => {
+    const bookDiv = document.createElement('div');
+    bookDiv.className = 'book';
+    const spanTitle = document.createElement('span');
+    spanTitle.textContent = book.title;
+    const spanAuthor = document.createElement('span');
+    spanAuthor.textContent = book.author;
+
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'btn';
+    removeBtn.textContent = 'Remove';
+    /* eslint-disable */
+    removeBtn.addEventListener('click', () => removeBook(index));
+    /* eslint-enable */
```

Comment:
- [ ] It's important to consider the benefits of keeping the linter enabled while coding. The linter serves as a valuable tool that aids us in maintaining clean and organized code. By providing real-time feedback and highlighting potential issues, it guides us towards best practices and consistent coding standards. Disabling the linter might seem like a shortcut, but it can lead to overlooked errors and compromise the quality of our codebase. Embracing the linter's suggestions not only improves code readability but also enhances collaboration within the team. So, let's keep the linter active to ensure our code remains robust and professional.